src/alert_actor.cc

```diff
@@ -299,8 +306,8 @@ alert_actor_test (bool verbose)
         NULL);
     assert (poller);
 
+    mlm_client_sendtox (rfc_evaluator, "agent-nut-alert", "rfc-evaluator-rules", "OK", NULL);
```

Comment:
TODO (in another PR) : convert strings like here into macros, like in other components